src/platform/qt/scripting/ScriptingController.cpp

```diff
@@ -39,6 +40,11 @@ ScriptingController::ScriptingController(ConfigController* config, QObject* pare
 		va_copy(argc, args);
 		QString message = QString::vasprintf(format, argc);
 		va_end(argc);
+		if (logger->p->m_config->getOption("logToStdout").toInt()) {
+			QTextStream out(stdout);
+			out << message << "\n";
+			out.flush();
```

Comment:
`LogController.cpp` uses a QT_VERSION check to make sure that `endl` is defined, but this equivalent implementation works in all Qt versions. Maybe we could apply the same change there to remove a conditional compilation?

Reply:
At the very least, IMO the `#define endl Qt::endl` should be replaced with `using Qt::endl;`.